contrib/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
```

Comment:
The downside of `bash` instead of `sh` is that it will make it harder to add OSX testing in the future without too much trouble(you probably remember https://github.com/bitcoin-core/secp256k1/pull/750)

Reply:
yeah, let me try again. :D